ChallengesWithTestsMarkVII/ChallengesSet05.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ChallengesWithTestsMarkVII
+{
+    public class ChallengesSet05
+    {
+        public int GetNextNumberDivisibleByN(int startNumber, int n)
+        {
+            for (int i = startNumber; i < 10000000; i++)
+            {
+                if (i % n == 0 && i > startNumber)
+                {
+                    return i;
+                }
+            }
+            return 0;
+        }
+
+        public void ChangeNamesOfBusinessesWithNoRevenueTo_CLOSED(Business[] businesses)
+        {
+            Business business = new Business();
+
+            for (int i = 0; i < businesses.Length; i++)
+            {
+                if (businesses[i].TotalRevenue == 0d)
+                {
+                    businesses[i].Name = "CLOSED";
+                }
+            }
+        }
+
+        public bool IsAscendingOrder(int[] numbers)
+        {
+            if (numbers == null || numbers.Length == 0)
+            {
+                return false;
+            }
+
+            for (int i = 0; i < numbers.Length - 1; i++)
+            {
+                if (numbers.Length == 1)
```

Comment:
This if-statement doesn't need to be checked repeatedly. Consider moving it prior to the loop, probably as an else-if to line 35